javascript/ql/test/query-tests/NodeJS/DubiousImport/externs.js

```diff
@@ -333,6 +333,15 @@ Array.prototype.push = function(var_args) {};
  */
 Array.prototype.reverse = function() {};
 
+/**
+ * Returns a new array with the elements in reversed order.
+ *
+ * @this {{length: number}}
+ * @template T
+ * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed
+ */
+Array.prototype.toReversed = function() {};
```

Comment:
Were these changes to `externs.js` needed.  

In production we extract a bunch of "externs", which are files that contain placeholder implementations for a good portion of the standard library.  
However, this file is just a partial copy of that for purposes of testing.  

So I think you can revert your change to this file.

Reply:
The following commit ef18a6e56265d7d5d2cf8501caab5cd0c797372a reverts the changes to [extern.js.](https://github.com/github/codeql/blob/ef18a6e56265d7d5d2cf8501caab5cd0c797372a/javascript/ql/test/query-tests/NodeJS/DubiousImport/externs.js#L1)